apps/forklift/runtime.exs

```diff
@@ -147,6 +147,9 @@ if System.get_env("COMPACTION_SCHEDULE") do
     ]
 end
 
+config :forklift,
+  overwrite_mode: !!System.get_env("OVERWRITE_MODE")
```

Comment:
I have concerns about the readability of this. Notnot in front of a direct environment variable call is a bit opaque. Consider a case block that clearly lays out the possible values of the variable and assigns a boolean value based on it

Reply:
204b775e9165b9715fe6f4e03f40816061e1ded6

Corrected this in the above commit